### PR TITLE
WRN-8067: Update PostCSS packages

### PR DESCRIPTION
**Pull request by hong6316**
### Issue Resolved / Feature Added
Some errors were found while `enact pack`

> Error: Cannot find module ‘sanitize.css/page.css’
Require stack:
/usr/local/lib/node_modules/@enact/cli/node_modules/postcss-normalize/dist/index.cjs.js
./node_modules/@enact/ui/Scroller/UiScroller.module.css
Error: Cannot find module ‘sanitize.css/page.css’
Require stack:
/usr/local/lib/node_modules/@enact/cli/node_modules/postcss-normalize/dist/index.cjs.js
./node_modules/@enact/ui/Scroller/Scroller.module.css
Error: Cannot find module ‘sanitize.css/page.css’
Require stack:
/usr/local/lib/node_modules/@enact/cli/node_modules/postcss-normalize/dist/index.cjs.js
./node_modules/@enact/sandstone/Dropdown/Dropdown.module.css

### Resolution
Update `postcss-normalize` to 10.0.1

### Additional Considerations
Update PostCSS packages because  `postcss-normalize` need `postcss` v8
(related to https://github.com/facebook/create-react-app/pull/10456)

### Links
WRN-8067

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

**Review by MikyungKim — COMMENTED** [review]
Please add a changelog.

File: package.json [hunk 1]
```diff
@@ -106,10 +106,11 @@
...
+    "postcss": "8.3.5",
+    "postcss-flexbugs-fixes": "5.0.2",
     "postcss-global-import": "1.0.6",
-    "postcss-loader": "3.0.0",
-    "postcss-normalize": "9.0.0",
+    "postcss-loader": "4.2.0",
+    "postcss-normalize": "10.0.1",
     "postcss-preset-env": "6.7.0",
     "postcss-resolution-independence": "1.0.0",
     "postcss-safe-parser": "5.0.2",
```
Comment (MikyungKim) [inline package.json:116]: Please remove "sanitize.css" from the list.

Comment (hong6316) [reply package.json:116]: ok

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+## unrelease
```
Comment (MikyungKim) [inline CHANGELOG.md:1]:
```suggestion
## unreleased
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+
+### pack
+
+* Removed `sanitize.css` dependency.
+* Updated CLI dependency of `postcss-normalize` to 10.0.1.
```
Comment (MikyungKim) [inline CHANGELOG.md:6]:
```suggestion
* Updated CLI dependency of `postcss` related modules including `postcss-normalize` 10.0.1. 
```

File: config/webpack.config.js [hunk 4]
```diff
@@ -117,32 +117,32 @@ module.exports = function (env) {
 				loader: require.resolve('postcss-loader'),
 				options: {
 					// https://webpack.js.org/guides/migrating/#complex-options
```
Comment (MikyungKim) [inline config/webpack.config.js:119]: let's remove this comment.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**